docs/contents/user-docs/authorization-and-access.md

```diff
@@ -0,0 +1,69 @@
+# Configure permissions for authorization and access
+
+There are two different Role-Based Access Control (RBAC) systems needed for ACK service controller authorization: Kubernetes RBAC and AWS IAM. 
+
+[Kubernetes RBAC][0] governs a Kubernetes user's ability to read or write Kubernetes resources, while [AWS Identity and Access Management][2] (IAM) policies govern the ability of an AWS IAM role to read or write AWS resources.
+
+[0]: https://kubernetes.io/docs/reference/access-authn-authz/authorization/
+
+!!! note "These two RBAC systems do not overlap"
+    The Kubernetes user that makes a Kubernetes API call with `kubectl` has no association with an IAM role. Instead, the IAM role is associated with the [service account][1] that runs the ACK service controller's pod.
+
+[1]: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+
+Refer to the following diagram for more details on running a Kubernetes API server with RBAC authorization mode enabled.
+
+![Authorization in ACK](../images/authorization.png)
+
+[2]: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+
+You will need to configure Kubernetes RBAC and AWS IAM permissions before using ACK service controllers.
+
+## Step 1: Configure Kubernetes RBAC
+
+As part of installation, Kubernetes roles are automatically created. These roles contain permissions to modify the Kubernetes custom resources (CRs) that the ACK service controller is responsible for.
+
+!!! note "Resources are namespace-scoped"
+    All Kubernetes CRs managed by an ACK service controller are namespace-scoped resources. There are no cluster-scoped ACK-managed CRs.
+
+By default, the following Kubernetes role resources are created when installing an ACK service controller:
+
+* `ack-$SERVICE-writer`: a `Role` used for reading and mutating namespace-scoped CRs that the ACK service controller manages.
+* `ack-$SERVICE-reader`: a `Role` used for reading namespaced-scoped CRs that the ACK service controller manages.
+
+If you already have an ACK service controller installed, the Kubernetes role might already exist. In this case, permissions to manage custom resource definitions (CRDs) and CRs associated with a newly-installed ACK service controller are added to the existing role.
+
+To rename a default Kubernetes role, modify the Kubernetes manifests that are used as part of the installation process. 
```

Comment:
Probably need a path for Helm as well - we wouldn't expect users to modify Helm charts before they're applied, and we don't support overriding this value in the `values.yaml` file, yet.